bower.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "health-facilities",
+  "version": "1.0.0",
+  "dependencies": {
+    "agrc": "agrc-widgets#4.0.0",
+    "agrc-jasmine-matchers": "~0.1.0",
+    "bootstrap": "^3.3",
```

Comment:
3.3.7 adds jquery 3.1 which breaks the dojo build. Either check out what I had to do in the [boilerplate pr](https://github.com/agrc/AGRCJavaScriptProjectBoilerPlate/pull/175) to make that work or pin the version to 3.3.6